crates/macro/src/html_tree/html_tag/mod.rs

```diff
@@ -153,7 +153,7 @@ impl ToTokens for HtmlTag {
             #vtag.add_attributes(vec![#((#attr_labels.to_owned(), (#attr_values).to_string())),*]);
             #vtag.add_listeners(vec![#(::std::boxed::Box::new(#listeners)),*]);
             #vtag.add_children(vec![#(#children),*]);
-            ::yew::virtual_dom::VNode::VTag(#vtag)
+            ::yew::virtual_dom::VNode::VTag(Box::new(#vtag))
```

Comment:
I think `VNode::from` is better. Also, for future reference, you need to use proper macro hygiene here. So this would be `::std::boxed::Box::new(#vtag)` in order to avoid collisions with the user namespace.